modules/integration-tests/src/main/java/org/apache/fluo/integration/client/LoaderExecutorIT.java

```diff
@@ -113,4 +139,18 @@ public void testSubmit() throws Exception {
       // expected from future
     }
   }
+
+  @Test
+  public void testSubmitResult() throws Exception {
+
+    LoaderExecutor le = client.newLoaderExecutor();
+    // Each time the CounterLoader is executed, it increments a counter and makes that value
+    // available
+    // in the returned Future
+    CompletableFuture<Loader> future = le.submit(new CounterLoader());
+    Assert.assertEquals(((CounterLoader) future.get()).getNewValue().longValue(), 1);
+
+    future = le.submit(new CounterLoader());
```

Comment:
This could call the other submit method that takes an id to get coverage on both.